code/game/objects/structures.dm

```diff
@@ -8,6 +8,8 @@
 	var/climbable = FALSE
 	var/mob/living/structureclimber
 	var/broken = 0 //similar to machinery's stat BROKEN
+	layer = BELOW_OBJ_LAYER

```

Comment:
I agree with frames being no brainer but this changes a lot of things. Are you sure no structure actually expects this.

Reply:
I'm sure, yeah. I've gone through the structures, and the only thing that stood out was mineral doors which should be on the airlock layer anyhow. I'll remove some now unnecessary layer changes from a few of them though.